aries/aries_vcx/src/handlers/issuance/issuer.rs

```diff
@@ -241,8 +242,18 @@ impl Issuer {
             revocation_info.rev_reg_id,
             revocation_info.tails_file,
         ) {
+            let rev_reg_delta_json = ledger
+                .get_rev_reg_delta_json(&rev_reg_id, None, None)
```

Comment:
I would probably prefer using a custom struct as well, although in the code review context this could also be solved by GitHub doing some static analysis and providing signature help (which an IDE does give you):
![image](https://github.com/hyperledger/aries-vcx/assets/11699479/a2c69e8d-1f34-44de-8da3-74311bc8ae2a)

Reply:
The thing is, even if I know the args are None, None, it's not clear what the default are, what do these None represent. Both are set in different places in code. But that's just a note, lets mark this resolved